cpp/src/copying/copy_range.cu

```diff
@@ -122,6 +111,16 @@ struct out_of_place_copy_range_dispatch {
 
     return p_ret;
   }
+
+  template <typename T, typename... Args>
+  std::enable_if_t<not cudf::is_rep_layout_compatible<T>(), std::unique_ptr<cudf::column>>
+  operator()(Args...)
+  {
+    if constexpr (std::is_same_v<T, cudf::struct_view>) {
+      CUDF_FAIL("list_view type not supported");
```

Comment:
Why does `struct_view` get its own fail message? 
And why does it say `list_view` not supported?

Reply:
The error is I should have been matching on ` cudf::list_view`. It original had a separate error message ( removes line 223 ) which I presumed we wanted to keep

Reply:
I would recommend just going with the default message here. I think it would be confusing to have a different message for one type. Is there a test looking for this message? If so, I think the test should be updated with this message instead.

Reply:
The pattern of struct view and list view have unique error messages occurs in multiple places so I presumed it was intentional. 

But I can remove them if they aren't needed now.

Reply:
Removed